hack/local-audit.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# About: This script runs the auditor locally in --verbose mode.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)
+
+# Print the usage for this script.
+usage() {
+    echo >&2 "Usage: $0 <SERVICE_ACCOUNT_KEY_FILE>"
+    exit 1
+}
+
+# Program entrypoint.
+main() {
+    # Ensure correct number of runtime params.
+    if [[ $# != 1 ]]; then
+        echo >&2 "Key-file not found."
+        usage
+    fi
+
+    # Define the service account key-file.
+    export GOOGLE_APPLICATION_CREDENTIALS=$1
+
+    # Build CIP binary.
+    make --makefile="${REPO_ROOT}/Makefile" build
```

Comment:
You can do

```
pushd "${REPO_ROOT}"
make build
popd
```

which looks a bit simpler to me.

Reply:
Ack